addons/website_sale/models/sale_order.py

```diff
@@ -237,6 +237,18 @@ def _get_non_delivery_lines(self):
     def _get_amount_total_excluding_delivery(self):
         return sum(self._get_non_delivery_lines().mapped('price_total'))
 
+    def _get_confirmation_template(self):
+        """
+        Override: If the order comes from a website that has a
+        different order confirmation template, use that instead of the global default.
```

Comment:
Would shorten this one
```suggestion
        Override: Use the website specific order confirmation email template if set.
```